doc/lsst.ap.pipe/getting-started.rst

```diff
@@ -0,0 +1,90 @@
+directly interface with data files. `lsst.ap.pipe` is no exception.
+The process of turning a set of raw data files and corresponding calibration
+products into a format the Butler understands is called ingestion. Ingestion
+can be somewhat camera-specific, and is outside the scope of the AP Pipeline.
+
+A utility to ingest data before running `lsst.ap.pipe` 
+is available in `lsst.ap.verify.ingestion`. Alternately, you may use a pre-
+ingested dataset or manually ingest files yourself following the directions
+for a given ``obs_`` package, e.g.,
+step 4 of `the obs_decam README <https://github.com/lsst/obs_decam/blob/master/README.md>`_.
+
+A standard ingestion workflow for DECam looks something like
+
+.. prompt:: bash
+
+   ingestImagesDecam.py input_loc --filetype raw path/to/raw/files
+   ingestCalibs.py input_loc --calib calib_loc path/to/flats/and/biases --validity 999
+   ingestCalibs.py input_loc --calib calib_loc --calibType defect --mode=skip path/to/defects --validity 0
+
+
+.. _section-required-data-products:
+
+Required data products
+======================
+
+For the AP Pipeline to successfully process data, the following is required:
+
+- **Raw science images** and `reference catalogs 
+  <https://community.lsst.org/t/creating-and-using-new-style-reference-catalogs/1523>`_
+  ingested into a main Butler repository
+
+  - The reference catalogs must be in a directory called `ref_cats` with subdirectories
+    called `gaia` and `pan-starrs` containing the appropriate catalog shards
+    
+- **Calibration products** (biases, flats, and defects, if applicable)
+  ingested into a Butler repository you must specify with the ``--calib`` flag on
+  the command line at runtime
+  
+  - To check if this requirement has been satisfied, you can inspect the
+    `calibRegistry.sqlite3` created in this repository and ensure the information
+    in the flat, bias, and defect tables is accurate
+    
+- **Template images** (of type ``deepCoadd`` by default) for difference imaging
+  must be either in the main Butler repository or in another location you may
+  specify with the ``--template`` flag on the command line at runtime
+
+A sample dataset from the `DECam HiTS survey <http://iopscience.iop.org/article/10.3847/0004-637X/832/2/155/meta>`_ 
+that works with ``ap_pipe`` is available as `ap_verify_hits2015 
+<https://github.com/lsst/ap_verify_hits2015>`_. However, this dataset must be
+ingested as described in :ref:`section-ingesting-data-files`, and the reference
+catalog and defect files must be decompressed and extracted.
+
+.. warning::
+
+   `lsst.ap.pipe` has only been used on DECam (and, correspondingly, with
+   `lsst.obs.decam`) so far, and does not yet support data from other cameras.
+   This functionality is coming soon
+   (see `DM-12315 <https://jira.lsstcorp.org/browse/DM-12315>`_).
+
+Please continue to :doc:`Pipeline Tutorial <pipeline-tutorial>` for more
+details about running the AP Pipeline and interpreting the results.
```

Comment:
I borrowed some of this style from the ["Getting Started" pipelines tutorial pages](https://pipelines.lsst.io/getting-started/index.html), since it presently exists in a somewhat odd space between topic-based docs and a, well, tutorial. 🤷‍♀️